src/app/pages/OnDemandAudioPage/OnDemandAudioPage.jsx

```diff
@@ -45,6 +48,20 @@ const StyledGelWrapperGrid = styled(GelPageGrid)`
   }
 `;
 
+const StyledGridItemParagraph = styled(Grid)`
+  @media (min-width: 22.5625rem) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
```

Comment:
I don't why we are going with 361px here, but I believe this should also be `22.5rem`